lib/winston/create-logger.js

```diff
@@ -76,9 +76,10 @@ module.exports = function (opts = {}) {
       if (args.length === 1) {
         const [msg] = args;
         const info = msg && msg.message && msg || { message: msg };
-        info.level = info[LEVEL] = level;
-        self._addDefaultMeta(info);
-        self.write(info);
+        const cloneInfo = Object.assign(Object.create(info), info);
```

Comment:
Is assign needed in addition to create here, where the second parameter to `assign` is the same as the parameter to `create`?

Reply:
Thanks @wbt 

## Is assign needed in addition to create here ?

A - Yes

## Where the second parameter to assign is the same as the parameter to create ?

consider you have `info` is an instance of `Error`

In ` Object.assign(Object.create(info), info) `  where ` Object.create `  is responsible for creating a deep copy of the prototype of `info` and ` Object.assign `  will overwritten by properties of `info`